src/blockdata/block.rs

```diff
@@ -152,16 +152,31 @@ impl BlockHeader {
 
     /// Computes the target [0, T] that a blockhash must land in to be valid
     pub fn target(&self) -> Uint256 {
+        Self::u256_from_compact_target(self.bits)
+    }
+
+    /// Computes the target value in Uint256 format, from a compact representation.
```

Comment:
I think if you make that `[Uint256]`, rustdoc can autogenerate links.

Reply:
I linked it manually, since the auto-linking [still isn't merged](https://github.com/rust-lang/rust/issues/43466).